### ISSUE TITLE
npm run watch doesn't work for Sitecore

### ISSUE THREAD

**Issue by blipson89** [connectivedx/fuzzy-chainsaw#150]
running `npm run watch` does not compile assets for the Sitecore site. I end up with:
![image](https://cloud.githubusercontent.com/assets/4602966/21718741/5e538444-d3e7-11e6-86a6-ec60a974cf66.png)

`npm run build` works perfectly fine, however.

**Comment by krambuhl** [comment]
`npm run watch` is used to develop fed static assets in the `localhost:8080` website, it writes files to a virtual filesystem – so this outcome is expected.   

You can add the `--watch` flag to a build script in package.json if you want to watch with a real filesystem. 

```
"scripts": {
   "webpack-build-watch": "webpack --config ./build/webpack/webpack.config.js --watch"
}
```

**Comment by blipson89** [comment]
would adding this flag this affect @pgregorova, who still uses the `localhost:8080` site? Everence uses Legendary Pancake, and `lp watch` will trigger the watch task for Fuzzy, which blows away all the assets.

**Comment by krambuhl** [comment]
If you need LP to support watching using the same command, it would affect @pgregorova's `npm run watch` script.   LP and FED resources should probably call different watch scripts.  

I'd like to keep `npm run watch` for our FC watch task, because FC will not always be integrated with LP. so maybe `npm run integration-watch` could be called from LP scripts – that doesn't exist now: PR away!

```
"scripts": {
    "watch": "gulp watch",
   "integration-watch": "webpack --config ./build/webpack/webpack.config.js --watch"
}
```

**Comment by kamsar** [comment]
`watch` isn't really a traditional `watch` in FC. It's more a `run` or `server` task imo

**Comment by krambuhl** [comment]
I think the true answer to this is issue to rename our `watch` to `npm run dev` and to change the `integration-watch` command to `npm run watch`.  I don't want to change the developer api at this moment, we will wait to make script api changes until we upgrade to webpack 2 (FC1.1~1.2)

**Comment by kamsar** [comment]
I have merged #152 into my current project and it seems to be working well. I altered the task name to `watch` while merging so it conforms to the LP task interface.

**Comment by krambuhl** [comment]
Going to close this issue for now. #153 contains a task for renaming this new task.
